pkg/parsers/operatingsystem/operatingsystem_linux.go

```diff
@@ -24,6 +24,11 @@ func GetOperatingSystem() (string, error) {
 	}
 	if i := bytes.Index(b, []byte("PRETTY_NAME")); i >= 0 {
 		b = b[i+13:]
```

Comment:
This can still panic right here if `b` is less than 13 characters (e.g. `PRETTY_NAME`, with nothing after it)